utils_cv/detection/model.py

```diff
@@ -0,0 +1,283 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import numpy as np
+from typing import List, Tuple, Union
+from pathlib import Path
+from functools import partial
+
+from PIL import Image
+import cv2
```

Comment:
Any chance we can avoid using cv2? I have seen problems before getting opencv running on a webservice. Also, opencv's imread read in images in BGR order, all other packages in RGB, this is confusing and will almost definitely lead to silent bugs.

Reply:
yeah, i can update to use pillow, forgot that cv2 is a pain sometimes.